tests/js/jest.config.js

```diff
@@ -60,8 +60,9 @@ export default {
   coverageReporters: ['lcov'],
   coverageDirectory: '<rootDir>/build/logs',
   collectCoverageFrom: [
-    '<rootDir>/assets/src/**/*.js',
+    '<rootDir>/packages/**/*.js',
     '<rootDir>/bin/**/*.js',
+    '!<rootDir>/packages/fonts/**',
```

Comment:
@swissspidy, I updated these, since we weren't actually collecting any coverage from anything (as all the code in `packages` were ignored). I had to exclude fonts though, as Istanbul for some reason didn't like it.

Is coverage still collected somewhere? Is there a report for each PR?

Reply:
Whoops! Good catch.

Do you recall the specific issue with the fonts package? We do have some tests in there, so it would be nice to include them in coverage.

Looks like the codecov config ought to ignore the `cli.js` script in there though.

Code coverage is still collected. There should have been a comment on the PR about it actually 🤔 

Direct link is https://app.codecov.io/gh/google/web-stories-wp though

Reply:
I think it was only `cli.js`, that failed - something about having `await` in non-`async` function?

Reply:
Cool, then let's only ignore `cli.js` here.